lib/thinreports/generator/pdf/document/font.rb

```diff
@@ -52,7 +52,7 @@ def setup_fonts
       # @param [String] file
       # @return [String] installed font name
       def install_font(name, file)
-        raise Errors::FontFileNotFound unless File.exists?(file)
```

Comment:
:memo: `File.exists?` is a deprecated method since 2.1.0.
http://docs.ruby-lang.org/ja/2.1.0/class/File.html#S_EXISTS--3F